procurement/models.py

```diff
@@ -179,3 +179,26 @@ def contract_time_remaining_desc(self):
             return "{} months left".format(int(timedelta.days / 30))
         else:
             return "{} days left".format(timedelta.days)
+
+
+class ClimateRepresentative(models.Model):
```

Comment:
This might want a
```
class Meta:
    abstract = True
```
in it otherwise you end up with a table for this plus tables for the inheriting classes which I'm not sure is what you want? There's some more details on this here: https://docs.djangoproject.com/en/4.1/topics/db/models/#model-inheritance

Reply:
Thanks, I missed this!